api/requirements.txt

```diff
@@ -3,11 +3,8 @@ fastapi[all]==0.52.0
 freezegun==0.3.15
 hnswlib==0.3.4
 httpx==0.12.1
-pyserini==0.8.1
+pygaggle==0.0.1
```

Comment:
Just to confirm, we don't need pyserini anymore because pygaggle requires it?

Reply:
Yeah that's why I took it out, but maybe I should keep it for clarity?